Classifiers/classifier-EWS_v2.json

```diff
@@ -1,4 +1,5 @@
 {
+	"releaseNotes": "-",
```

Comment:
I think that this should be documented in RN

Reply:
I'll let @kirbles19 know about it

Reply:
@JonathanMeler you simply need to document your change on this line ...